dataGen.js

```diff
@@ -100,1105 +12,43 @@ const genTime = () => {
 
 const commentDataGen = (string, songLength) => {
   const comments = [];
-
-  string = string.replace(/\.|,/g, '');
-
   let stringArr = string.split(' ');
-
-  for(let i = 1; i < 15; i++) {
-    const comment_id = 1;
+  
+  // construct comments array
+  for (let i = 1; i < 15; i++) {
```

Comment:
Is this producing the same 15 comments for each song? Does each song have commentID's from 1-15?

Reply:
Each song has 15 comments and their own set of comment ids. I added a songId to differentiate from the comment ids.

Reply:
Thanks for the clarification. I believe that works out.